recipes/libcurl/all/conanfile.py

```diff
@@ -213,13 +207,30 @@ def build_requirements(self):
         else:
             self.tool_requires("libtool/2.4.6")
             self.tool_requires("pkgconf/1.7.4")
-            if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
+            if self._settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", default=False, check_type=bool):
```

Comment:
```suggestion
            if self._settings_build.os == "Windows" and not self.conf.get("tools.microsoft.bash:path", default=False, check_type=str):
```
A path is a string, not a boolean. I know, there are other recipes using it, but they are incorrect too.